app/src/main/java/com/example/polina/socialnetwork/FormActivity.java

```diff
@@ -33,24 +33,20 @@
 import java.io.IOException;
 import java.io.InputStream;
 import java.net.URL;
-import java.text.ParseException;
-import java.text.SimpleDateFormat;
-import java.util.Calendar;
-import java.util.Date;
 ;
 
 
 @EActivity(R.layout.form_activity)
 public class FormActivity extends Activity {
-    public static final String NAME = "name";
+
     @ViewById(R.id.form_name)
-    public TextView twname;
+    public TextView tvName;
     @ViewById(R.id.form_birthday)
-    public TextView twbithday;
+    public TextView tvBithday;
     @ViewById(R.id.form_sex)
-    public TextView twsex;
+    public TextView tvSex;
     @ViewById(R.id.form_image)
-    public ImageView twimage;
+    public ImageView tvImage;
```

Comment:
why did you call it tvImage? don't use prefix like tv*, m*, tw*; just "image", our "userLogo"